### PR TITLE
docs: include examples in README.md

### PR DESCRIPTION
**Pull request by gdams**
We should include the usage examples from the manual. This also includes instructions on how to test citgm on a local module 

- [x] documentation is changed or added
- [x] contribution guidelines followed [here](CONTRIBUTING.md)
- [x] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,16 @@ Options:
   -a, --append                Turns on append results to file mode rather than replace
 ```
 
+### Examples:
+  Test the latest underscore module:
+    `citgm underscore@latest`
+
+  Test a local module:
```
Comment (MylesBorins) [inline README.md]: is testing a local module working right now? Can you test?

Comment (gibfahn) [inline README.md]: @MylesBorins turns out it is working, it was just a strange error message that was confusing me before (basically if you do a `citgm ./deps/npm` it fails unless you first `npm i -g rimraf`).

Comment (gdams) [inline README.md]: @MylesBorins it is fully working, perhaps we should add a test to citgm that tests this feature?

Comment (gdams) [inline README.md]: @MylesBorins test included here: https://github.com/nodejs/citgm/pull/336

**Pull request merged**